### ISSUE TITLE
custom mac address for jail

### ISSUE THREAD

**Issue by gregf** [cbsd/cbsd#159]
Is it possible to customize the mac address for a jail?

**Comment by olevole** [comment]
You can change it only for vnet/VIMAGE-based jail.

Unfortunately, I have not yet made possible to change this through a dialog or cli, but you can change it directly in SQLite3. For example, if jail - vnet1, select current MAC :

~~~
% sqlite3 ~cbsd/jails-system/vnet1/local.sqlite "SELECT nic_hwaddr FROM jailnic"
02:ff:f0:e8:b4:b1
~~~

To change it:
~~~
% sqlite3 ~cbsd/jails-system/vnet1/local.sqlite "UPDATE jailnic SET nic_hwaddr='02:ff:f0:e8:b4:b4'"
~~~

PS: Since CBSD 11.0.10, MAC address for vnet and bhyve is random-generated.

**Comment by gregf** [comment]
Thank you! Hope this makes it into the tui in the future.

**Comment by olevole** [comment]
Will be available in CBSD 11.0.15
